src/main/java/org/fcrepo/camel/FcrepoEndpoint.java

```diff
@@ -34,43 +34,8 @@
 @UriEndpoint(scheme = "fcrepo")
 public class FcrepoEndpoint extends DefaultEndpoint {
 
-    private String baseUrl = "";
-
-    @UriParam
-    private String contentType = null;
-
-    @UriParam
-    private String accept = null;
-
-    @UriParam
-    private String transform = null;
-
-    @UriParam
-    private String authUsername = null;
-
-    @UriParam
-    private String authPassword = null;
-
-    @UriParam
-    private String authHost = null;
-
-    @UriParam
-    private Boolean secure = false;
-
     @UriParam
-    private Boolean tombstone = false;
-
-    @UriParam
-    private Boolean metadata = true;
-
-    @UriParam
-    private Boolean throwExceptionOnFailure = true;
-
-    @UriParam
-    private String preferInclude = null;
-
-    @UriParam
-    private String preferOmit = null;
+    private FcrepoConfiguration configuration;
```

Comment:
For my own understanding, based on the following documentation, it would seem that the `configuration` field would now be annotated with `@UriParams` instead of `@UriParam`.
http://camel.apache.org/endpoint-annotations.html

Do you have any insight into why `@UriParam` is adequate here?